src/lib/structures/expr.ml

```diff
@@ -493,7 +493,7 @@ module SmtPrinter = struct
         Format.fprintf fmt "| %a@,"  (Util.print_list ~sep:"," ~pp:print) l;
       ) trs
 
-  and print_verbose fmt t = print fmt t
```

Comment:
This code is quiet strange. Are `print_verbose` and `print` equal now?

Reply:
The difference between `print_verbose` and `print_simple` is that `print_verbose` used to print the type as well as the expression, but since the printing of the types was removed not long ago, making `print_verbose` call `print` (which is just a dispatcher) leads to infinite loop.
So yeah, I don't know if we want to add back the printing of types, or just have one printing function, or print other information, but for now `print_verbose` and `print_simple` do the same thing. It's not great to have useless code but it's better than an infinite loop.